<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdTest</class>
 <widget class="QWidget" name="ThirdTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThirdTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Qt线程测试</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pbnThirdstar">
        <property name="text">
         <string>线程启动</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbnstar">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pbnStop">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pbnClear">
        <property name="text">
         <string>清空文本</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pbnThirdstop">
        <property name="text">
         <string>线程停止</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="labPic">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labsta">
        <property name="text">
         <string>当前线程状态：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
